sdk/communication/AzureCommunicationChat/Tests/Util/Recordings/test_CreateThread_WithoutParticipants.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
We need to remove the recording files until #897 is complete. This contains sensitive info.

Reply:
I've removed the JSON recording files